package.json

```diff
@@ -13,12 +13,12 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "coveralls": "^3.0.7",
-    "eslint": "^6.5.1",
+    "eslint": "^7.32.0",
     "eslint-config-google": "^0.14.0",
-    "ot-json0-v2": "ottypes/json0",
     "lolex": "^5.1.1",
     "mocha": "^8.2.1",
     "nyc": "^14.1.1",
+    "ot-json0-v2": "ottypes/json0",
```

Comment:
Interesting, why did we need to add this?

Reply:
Oh it just got moved when I ran `npm install` — I'd added this by hand before, and I guess I don't alphabet so good 🤦🏼‍♂️

Reply:
Oh right and I don't read diffs good